wikimedia/wikipedia-ios
# Talk page - link sharing and design adjustment

**Phabricator:** First item in https://phabricator.wikimedia.org/T310288#8389567
https://phabricator.wikimedia.org/T312314#8389813

### Notes
* This PR addresses small issues, an encoding issue causing links to break when shared, and a design adjustment on alerts

### Test Steps
1. From a talk page, click Share and choose the airdrop option. The link should be received and opened normally
2. Cause any alert to show - the icon should be bigger
